packages/api/src/api/project.ts

```diff
@@ -0,0 +1,47 @@
+import { Router } from 'express';
+import { Project } from '../entities/Project';
+import logger from '../logger';
+import { env } from '../env';
+
+export const project = Router();
+
+project.get('', async (req, res) => {
+    try {
+      const projectList = await req.entityManager.find(Project, {});
+      
+      const repoNodeIds = projectList.map(project => project.nodeID);
+
+      const fetchRes = await fetch('https://api.github.com/graphql', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          'Authorization': `bearer ${req.user?.githubToken}`,
```

Comment:
Only problem I am running into during tests is testing branching on "req.user?.githubToken". Is there any way to mock the session?

Reply:
On your test handler, you can pass a second middleware to modify the request. Should look like this

```typescript
const handler = await testHandler(yourHandler, (req, _res, next) => {
  req.user = { githubToken: 'token' };
  next();
});
```